# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/public/GeckoContentController.h
# Commit: 76279f0a6d33
# Full Hash: 76279f0a6d33d6fe128a9f3aabe5393e52181818
# Author: James Willcox <snorp@snorp.net>
# Date: 2020-06-08 21:38:11
# Regressor Bug: 1536833
# File Overlap Count: 1
# Description:
#   Bug 1536833 - Fix overscroll effect on Android r=botond,geckoview-reviewers,agi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78130
# ==============================================================================

diff -r ad757ab87752 -r 76279f0a6d33 gfx/layers/apz/public/GeckoContentController.h
--- a/gfx/layers/apz/public/GeckoContentController.h	Sun Jun 07 23:15:45 2020 +0000
+++ b/gfx/layers/apz/public/GeckoContentController.h	Mon Jun 08 17:13:28 2020 +0000
@@ -145,9 +145,11 @@
 
   virtual void CancelAutoscroll(const ScrollableLayerGuid& aGuid) = 0;
 
-  virtual void UpdateOverscrollVelocity(float aX, float aY,
+  virtual void UpdateOverscrollVelocity(const ScrollableLayerGuid& aGuid,
+                                        float aX, float aY,
                                         bool aIsRootContent) {}
-  virtual void UpdateOverscrollOffset(float aX, float aY, bool aIsRootContent) {
+  virtual void UpdateOverscrollOffset(const ScrollableLayerGuid& aGuid,
+                                      float aX, float aY, bool aIsRootContent) {
   }
 
   GeckoContentController() = default;